admin/Models/EvaluationStageDto.cs

```diff
@@ -0,0 +1,5 @@
+public class EvaluationStageDto
+{
+    public string action { get; set; }
+    public string dateTime { get; set; }
+}
```

Comment:
The pattern for other models is to wrap the class in the `Epa.Camd.Quartz.Scheduler.Models` namespace:
```suggestion
namespace Epa.Camd.Quartz.Scheduler.Models
{
    public class EvaluationStageDto
    {
        public string action { get; set; }
        public string dateTime { get; set; }
    }
}
```

Reply:
Updated per comment.